src/Apps/Order/Routes/Accept/index.tsx

```diff
@@ -225,15 +195,49 @@ export class Accept extends Component<AcceptProps, AcceptState> {
                       Submit
                     </Button>
                     <Spacer mb={2} />
-                    <ConditionsOfSaleDisclaimer />
+                    <ConditionsOfSaleDisclaimer textAlign="center" />
+                  </Media>
+                </Flex>
+              }
+              Sidebar={
+                <Flex flexDirection="column">
+                  <Flex flexDirection="column">
+                    <Media greaterThan="xs">
+                      {className => (
+                        <ArtworkSummaryItem
+                          className={className}
+                          order={order}
+                        />
+                      )}
+                    </Media>
+                    <ShippingSummaryItem order={order} locked />
+                    <CreditCardSummaryItem order={order} locked />
+                  </Flex>
+                  <Media greaterThan="xs">
                     <Spacer mb={2} />
-                    <Helper artworkId={artwork.id} />
-                  </>
-                </Media>
-              </Flex>
-            }
-          />
-        </HorizontalPadding>
+                  </Media>
+                  <Media at="xs">
+                    <>
+                      <Spacer mb={2} />
+                      <Button
+                        onClick={this.onSubmit}
+                        loading={isCommittingMutation}
+                        size="large"
+                        width="100%"
+                      >
+                        Submit
+                      </Button>
+                      <Spacer mb={2} />
+                      <ConditionsOfSaleDisclaimer />
+                      <Spacer mb={2} />
+                    </>
+                  </Media>
+                </Flex>
+              }
+            />
+          </HorizontalPadding>
+        </Box>
+        <StickyFooter orderType="OFFER" artworkId={artwork.id} />
```

Comment:
Since the helper is on every page, I wonder if it makes sense to move it (+ the padding-bottom wrapper) to the OrderApp component.